docker-compose.yml

```diff
@@ -41,7 +41,7 @@ services:
       POSTGRAPHILE_DB_URL: postgres://postgraphile:secret@engine-db:5432/engine
       POSTGRAPHILE_DB_USER: postgraphile
       POSTGRAPHILE_DB_SCHEMA: noumena
-      POSTGRAPHILE_TRUSTED_ISSUERS: "http://keycloak:11000/**,"
+      POSTGRAPHILE_ALLOWED_ISSUERS: http://keycloak:11000/realms/nplintegrations,http://keycloak:11000,http://localhost:11000/realms/nplintegrations
```

Comment:
nit: You might also keep values in multiple lines for readability, see [example](https://stackoverflow.com/a/57562964/14838177)